src/lib/isStrongPassword.js

```diff
@@ -78,6 +79,10 @@ function scorePassword(analysis, scoringOptions) {
   if (analysis.symbolCount > 0) {
     points += scoringOptions.pointsForContainingSymbol;
   }
+
+  // If password exceeds maximum configured length, password is no longer considered strong
+  points = analysis.length <= scoringOptions.maxLength ? points : 0;
+
```

Comment:
If password exceeds `maxLength` we should not consider it weak (it is exactly the opposite😄).